data/korath/korath ships.txt

```diff
@@ -270,6 +270,75 @@ ship "Korath Raider Mark XLII"
 	description `Constant warfare against surrounding aliens has been building pressure on Korath shipyards to recover some of their ancient prowess. Now, with a fresh industrial network to support their endeavors, the shipwrights of the Korath have updated the design to deal with modern targets.`
 
 
+ship "Korath Ravager"
+	sprite "ship/ravager"
+	thumbnail "thumbnail/ravager"
+	attributes
+		category "Heavy Warship"
+		"cost" 3760000
+		"shields" 32700
+		"hull" 9850
+		"required crew" 193
+		"bunks" 221
+		"mass" 1120
```

Comment:
This brings the acceleration to a little lower than the Shield Beetle instead of a little lower than a Model 512.
The turn rate is now only slightly lower than a Hurricane instead of a decent chunk less.
```suggestion
		"mass" 979
```